src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+interface IPaymentCoordinator {
+
+    /// STRUCTS ///
+	struct RangePayment {
+        IStrategy[] strategies;
+        // relative weights of shares in the strategies
+        uint256[] weights;
```

Comment:
IIRC we decided to do 1 array of StrategyParams, rather than 2 (less input validation required)

Reply:
we should call thsee multipliers

Reply:
addressed both in https://github.com/Layr-Labs/eigenlayer-contracts/pull/470/commits/1d438f42e18ff6e494ed572df71b024a4e965de6